.eslintrc.js

```diff
@@ -1,31 +1,255 @@
 module.exports = {
   env: {
     node: true,
+    es6: true,
+    'jest/globals': true,
   },
-  extends: 'eslint:recommended',
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:@typescript-eslint/recommended-requiring-type-checking',
+    'plugin:jest/recommended',
+    'prettier/@typescript-eslint', // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
+    'plugin:prettier/recommended',
+  ],
+  ignorePatterns: [
+    'e2e/__helpers__',
```

Comment:
can we move all the things inside this `ignorePatterns` to `.eslintignore` ?

Reply:
This is fixed in latest commit